src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -47,12 +47,60 @@ public Optional<Movie> get(Long id) {
     @Override
     public List<Movie> getAll() {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
-            CriteriaQuery<Movie> criteriaQuery = session.getCriteriaBuilder()
-                    .createQuery(Movie.class);
-            criteriaQuery.from(Movie.class);
-            return session.createQuery(criteriaQuery).getResultList();
+            return session.createQuery("from Movie").list();
         } catch (Exception e) {
-            throw new DataProcessingException("Can't get all movies", e);
+            throw new DataProcessingException("Can't get Movies", e);
         }
     }
+
+    @Override
+    public boolean update(Movie movie) {
```

Comment:
Consider changing the return type of the `update` method to `void` or `Movie`. The current implementation returns a boolean based on equality check, which might not be reliable or meaningful.